Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -377,6 +377,15 @@ class BocList
     selectedRows.OnSelectionChanged (bocList, false);
   }
 
+  public static OnCustomCheckboxClick (elementName: string): void
```

Comment:
since I noticed this in the repo:

I was referring to integrating this code with the rest of the click-handling code already available in BocList.ts for setting the checkbox/radiobutton. We do not need any Javascript-Hookup in the C# code behind. We just need to wire up the relevant DOM elements. 

There is also code in BocList.ts that makes the entire cell work as the checkbox so maybe we just need to make the select element click "transparent"?